lib/ts/recipe/emailpassword/components/library/input.tsx

```diff
@@ -20,7 +20,6 @@ import CheckedIcon from "../../../../components/assets/checkedIcon";
 import ErrorIcon from "../../../../components/assets/errorIcon";
```

Comment:
if the label is `""` for a form field, and it's necessary, do not show the `*` char